kind/images/node/README.md

```diff
@@ -0,0 +1,19 @@
+<!--TODO(bentheelder): fill this in much more thoroughly-->
+# images/node
+
+This directory contains sources for building the `kind` "node" image.
+
+These are included in [`pkg/build/sources`](./../../pkg/build/sources), 
```

Comment:
there's no pkg/build/sources?

Reply:
Oop, that's one of the things I dropped to trim down the PR. I'll update the comment.

Reply:
fixed